pom.xml

```diff
@@ -61,8 +61,10 @@
         <junit.version>5.9.0</junit.version>
         <lombok.version>1.18.30</lombok.version>
         <lombok-maven-plugin.version>1.18.20.0</lombok-maven-plugin.version>
-        <hadoop.version>3.4.0</hadoop.version>
+        <hadoop.version>3.3.6</hadoop.version>
         <hudi.version>0.14.0</hudi.version>
+        <aws.version>2.28.22</aws.version>
+        <hive.version>2.3.9</hive.version>
```

Comment:
There is a 4.X.X release, can we try using that or at least the 3.X.X line? I am worried that any vulnerability patches will not make it to the N-2 major version line.